### PR TITLE
P0 completed for ECom macro project

### PR DESCRIPTION
**Pull request by hrishikeshmane**

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -27,13 +26,17 @@
...
+							<p>Sign in</p>
+							<p class="font-bold text-sm">Account & Lists</p>
+						</a>
 					</div>
 
 					<div class="link white pointer mx-2">
-						<p>Wishlist	</p>
-						<p class="font-bold text-sm">& Orders</p>
+						<a href="./wishlist.html">
+							<p>Wishlist	</p>
```
Comment (Aditi0505) [inline cart.html]: Suggestion: Please use a tool for code formatting. This will take care of the extra spacing too.

File: index.html [hunk 2]
```diff
@@ -61,7 +64,7 @@
 			</div>
 		</header>
 
-		<main>
+		<main class="bg-light-gray">
 			<img src="./assets/banner.jpg" class="w-full object-fit-cover m-auto" />
```
Comment (Aditi0505) [inline index.html]: Suggestion: Please provide alt for accessibility checkpoint.

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+			</div>
+		</header>
+
+		<main class="products h-full">
+            <h2 class="flex flex-center mt-5">Your Wishlist</h2>
+			<div class="grid gap-10 grid-cols-4 product-feed absolute mt-5 mx-10 z-4"> 
+                <div class="card card-shadow relative flex flex-col m-5 bg-white rounded-sm border p-10">
+                    <p class="absolute top-1 left-2 text-xs italic">Men's Clothing</p>
+                    <i class="fa fa-heart fa-2x absolute top-1 right-1 pointer red" aria-hidden="true"></i>
+                    <img src="./assets/tshirt.jpg" height="200" width="200" class="object-fit-cover m-auto" />
```
Comment (Aditi0505) [inline wishlist.html]: Suggestion: Please provide alt for accessibility checkpoint.

File: wishlist.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                        <h4>₹ 1,338.00</h4>
+                    </div>
+                    <button class="button bg-amazon_yellow bg-amazon_yellow--hover p-1 rounded-sm">
+                        Add to Cart
+                    </button>
+                </div>
+
+                <div class="card card-shadow relative flex flex-col m-5 bg-white rounded-sm border p-10 ">
+                    <p class="absolute top-1 left-2 text-xs italic">Men's Clothing</p>
+                    <img src="./assets/tshirt2.jpg" height="200" width="200" class="object-fit-cover m-auto" />
```
Comment (Aditi0505) [inline wishlist.html]: Suggestion: Please provide alt for accessibility checkpoint.

File: wishlist.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                    </div>
+                    <button class="button bg-amazon_yellow bg-amazon_yellow--hover p-1 rounded-sm">
+                        Add to Cart
+                    </button>
+                </div>
+
+                <div class="card card-shadow relative flex flex-col m-5 bg-white rounded-sm border p-10 ">
+                    <p class="absolute top-1 left-2 text-xs italic">Men's Clothing</p>
+                    <i class="fa fa-heart fa-2x absolute top-1 right-1 pointer red" aria-hidden="true"></i>
+                    <img src="./assets/jacket.jpg" height="200" width="200" class="object-fit-cover m-auto" />
```
Comment (Aditi0505) [inline wishlist.html]: Suggestion: Please provide alt for accessibility checkpoint.

File: wishlist.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+                    </div>
+                    <button class="button bg-amazon_yellow bg-amazon_yellow--hover p-1 rounded-sm">
+                        Add to Cart
+                    </button>
+                </div>
+
+                <div class="card card-shadow relative flex flex-col m-5 bg-white rounded-sm border p-10 ">
+                    <p class="absolute top-1 left-2 text-xs italic">Women's  Clothing</p>
+                    <i class="fa fa-heart fa-2x absolute top-1 right-1 pointer red" aria-hidden="true"></i>
+                    <img src="./assets/top1.png" height="200" width="200" class="object-fit-cover m-auto" />
```
Comment (Aditi0505) [inline wishlist.html]: Suggestion: Please provide alt for accessibility checkpoint.

**Review by Aditi0505 — APPROVED** [review]
Left small suggestions. Rest looks good to me.
Also, please provide description and screenshots for better visualization.

**Review by Creator-singh — APPROVED** [review]
looks good to me

**Pull request merged**